doc/api/errors.markdown

```diff
@@ -243,8 +243,9 @@ by other contexts.
 <!--type=misc-->
 
 A JavaScript "exception" is a value that is thrown as a result of an invalid operation or
```

Comment:
Nit: The scare quotes around `exception` are unnecessary IMO. Since we're editing nearby text, maybe this is a good opportunity to remove them. Totally optional, feel free to ignore, I'm fine with the change as is.